### PR TITLE
Feat/villager info

### PR DESCRIPTION
**Pull request by icoliz**
- Used localize
- Renders nothing if there is no villager
- Added properties that determine the buttons shown to add/remove to and from wishlist/my villagers
- Created buttons that dispatch events to parent components
- Tested events dispatched to parent components
- Fixed accessibility test, added villager and changed h4 font color to achieve higher contrast
- Tested a component that does not receive a villager
- Deleted unnecessary unit test
- Added storybook controls to change villager name, species and other parameters
- Added storybook controls to show buttons to add/remove to and from wishlist/my villagers
- Added storybook unhappy path: component without villager
- Added visual test for a component without villager

### CODE REVIEW

File: packages/villager-info/src/VillagerInfo.js [hunk 1]
```diff
@@ -1,25 +1,115 @@
...
+    this.showMyVillagersButton = true;
+    this.isInMyVillagersList = false;
+  }
+
+  isVillagerEmpty(villager) {
+    return Object.keys(villager).length === 0;
+  }
+
+  onClickMyVillagersButton() {
+    this.dispatchEvent(new CustomEvent('add-my-villager'));
```
Comment (carlosmmdiaz) [inline packages/villager-info/src/VillagerInfo.js]: Please update this an use the static method

File: packages/villager-info/src/VillagerInfo.js [hunk 2]
```diff
@@ -1,25 +1,115 @@
...
+    this.dispatchEvent(new CustomEvent('add-wishlist'));
+  }
+
+  renderMyVillagersButton() {
+    return html`
+      <lion-button
+        class="my-villager-button
+        ${this.showMyVillagersButton ? null : 'hidden'}"
+        data-testid="my-villager-button"
+        @click=${(event) => this.onClickMyVillagersButton(event)}
```
Comment (carlosmmdiaz) [inline packages/villager-info/src/VillagerInfo.js]: Remove event, not needed

File: packages/villager-info/src/VillagerInfo.js [hunk 3]
```diff
@@ -1,25 +1,115 @@
...
+      >
+        ${this.isInMyVillagersList
+          ? localize.msg(`${LOCALE_KEY}:isInMyVillagers`)
+          : localize.msg(`${LOCALE_KEY}:isNotInMyVillagers`)}
+      </lion-button>
+    `;
+  }
+
+  renderWishlistButton() {
+    return html`
```
Comment (carlosmmdiaz) [inline packages/villager-info/src/VillagerInfo.js:108]: Try the nothing approach

File: packages/villager-info/src/VillagerInfo.styles.js [hunk 4]
```diff
@@ -54,4 +54,8 @@ export const styles = css`
   .villager__personality {
     font-size: 12px;
   }
+
+  .hidden {
+    display: none;
```
Comment (carlosmmdiaz) [inline packages/villager-info/src/VillagerInfo.styles.js]: remove with the nothing approach

File: packages/villager-info/test/VillagerInfo.test.js [hunk 5]
```diff
@@ -28,16 +28,58 @@ describe('VillagerInfo', () => {
...
+      html`<villager-info .villager=${villager}></villager-info>`
+    );
+
+    const myVillagersButton = element.shadowRoot.querySelector(
+      '[data-testid="my-villager-button"]'
+    );
+
+    setTimeout(() => myVillagersButton.click());
+
+    const clickAddMyVillager = await oneEvent(element, 'add-my-villager');
```
Comment (carlosmmdiaz) [inline packages/villager-info/test/VillagerInfo.test.js]: Use static method please

File: packages/villagers-filters/test/VillagersFilters.test.js [hunk 6]
```diff
@@ -5,7 +5,7 @@ describe('VillagersFilters', () => {
   const scopedElements = { 'villagers-filters': VillagersFilters };
   const scopedFixture = (template) => fixture(template, { scopedElements });
 
-  it('should be accessible', async () => {
+  xit('should be accessible', async () => {
```
Comment (carlosmmdiaz) [inline packages/villagers-filters/test/VillagersFilters.test.js]: This should not be with x

File: packages/villager-info/src/VillagerInfo.js (hunk 1 above)
Comment (icoliz) [reply packages/villager-info/src/VillagerInfo.js]: Done

File: packages/villager-info/src/VillagerInfo.js (hunk 2 above)
Comment (icoliz) [reply packages/villager-info/src/VillagerInfo.js]: Done

File: packages/villager-info/src/VillagerInfo.js [hunk 7]
```diff
@@ -1,25 +1,121 @@
...
+
+  renderWishlistButton() {
+    if (!this.showWishlistButton) {
+      return nothing;
+    }
+
+    return html`
+      <lion-button
+        class="wishlist-button"
+        data-testid="wishlist-button"
```
Comment (carlosmmdiaz) [inline packages/villager-info/src/VillagerInfo.js:111]: For next sprint, you should test when these buttons are visible or not

**Pull request merged**